homework-submission/week1/App.js

```diff
@@ -0,0 +1,102 @@
+// the role of this component is to create a list tag
+// with three prop items.(description, deadline and class for CSS)
+// this component is used both for static and dynamic
+
+const List = ({ description, deadline, className }) => {
+  return (
+    <li className={className}>
+      {description}, {deadline}
+    </li>
+  );
+};
+
+//STATIC LIST
+
+// this component is used for creating a <ul> tag to contain
+// <li> tags
+
+const UnorderedStaticList = () => {
+  return (
+    <ul>
+      <List description="Get out of bed" deadline="Wed Sep 13 2017" className="true" />
```

Comment:
A `true/false` are a little confusing for a developer who is trying to reach out the css files and figuring out what belongs to whom.
A css name like would be more convenient as a className by any case.

Reply:
👍